nibabel/viewers.py

```diff
@@ -61,6 +61,9 @@ def __init__(self, data, affine=None, axes=None, title=None):
         title : str or None, optional
             The title to display. Can be None (default) to display no
             title.
+        vlim : array-like or None, optional
```

Comment:
Sure - I agree that the ``ylim == clim thing`` is bad in the default case, the question is what to do in the not-default case. 

For example - I guess that people will want to use `vlim` to get better contrast in the images, but they don't intend to change the time-series pane.   Do you think that's right?  In which case, should `ylim` be another input argument, that defaults to the original heuristic, regardless of `vlim`?